source/features/sticky-sidebar.css

```diff
@@ -35,6 +35,11 @@ Exclusively use simple sums and use `0px` instead of `0`
 .rgh-sticky-sidebar {
 	position: sticky;
 	top: var(--rgh-sticky-sidebar-offset, 0) !important;
+	/*
+		Make sure the container's z-index is higher than the readme header
+		to avoid the README header overlapping the sidebar.
+	 */
+	z-index: 9;
```

Comment:
Rather than trigger another z-index fight, I'd either disable the feature entirely on the repo home or disable it when that modal is open. Hopefully this can be done without using `:has`